main_window/main_window.py

```diff
@@ -14,14 +14,44 @@
 #     pyside6-uic form.ui -o ui_form.py, or
 #     pyside2-uic form.ui -o ui_form.py
 from .ui import Ui_Widget
-
+ 
 points = np.empty((0,2))
 
 @dataclass
 class PlotInfo:
     points: np.array
     data_line: PlotDataItem
 
+class Actuators:
+    def __init__(self, parentGrid):
+        valves = {}
+        self.valves = {}        
+        valves[0] = TelemetryLabel("Fire Valve", "CLOSED", 0, 2, parentGrid)
+        valves[13] = TelemetryLabel("Quick Disconnect", "CLOSED", 0, 0, parentGrid)
+        valves[14] =  TelemetryLabel("Igniter", "CLOSED", 0, 4, parentGrid)
+        for i in range(1, 13):
+            #There will be three label at each row, therefore divide by three, add 1 to skip the first row of valves
+            #Row timed 2 because there will be two label for state and for the name
+            valves[i] = TelemetryLabel("XV-" + str(i), "CLOSED", ((i - 1)// 3) + 1 , ((i - 1) % 3) * 2, parentGrid)
+
+    def turnOnValve(id):
+        valves[id].changeState("OPEN")
+
+    def turnOffValve(id):
+        valves[id].changeState["CLOSED"]   
+
+class TelemetryLabel:
+    def __init__(self, name, state, row, column, parentGrid):
+        self.row = row
+        self.column = column
+        self.qName =  QLabel(name)
+        self.qState = QLabel(state)
+        parentGrid.addWidget(self.qName, row, column)
+        parentGrid.addWidget(self.qState, row, column + 1)
```

Comment:
If you're able, can you set the style of these labels to match the previous style?